WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -551,6 +551,27 @@ final class EditableOrderViewModel: ObservableObject {
         resetAddressForm()
     }
 
+    func addTaxRateAddressToOrder(taxRate: TaxRate) {
+        guard let taxBasedOnSetting = taxBasedOnSetting else {
+            return
+        }
+
+        let address = Address.from(taxRate: taxRate)
+        let input: OrderSyncAddressesInput
+        switch taxBasedOnSetting {
+        case .customerBillingAddress:
+            input = OrderSyncAddressesInput(billing: address, shipping: nil)
+        case .customerShippingAddress:
+            input = OrderSyncAddressesInput(billing: nil, shipping: address)
+        default:
```

Comment:
Should we use `case .shopBaseAddress` explicitly rather than `default`? So if we happen to add new cases we'll be warned on compile time that something is unhandled:
```suggestion
        case .shopBaseAddress:
```

Reply:
In this case `OrderSyncAddressesInput` has only two properties: billing, and shipping, and we included these cases in the switch. Even if we added more cases, they wouldn't be addressable for our specific case here. But yours is a good point, and a good generic guideline.